doc/source/serve/deploy-many-models/multi-app.md

```diff
@@ -168,11 +168,11 @@ $ serve run config.yaml
 
```

Comment:
Merge serve run section with serve deploy? Should we point out the serve run application change here

Reply:
I think it can still be useful to mention it here. I've cut down on the section so it's less repetition, and more of just pointing out you can use serve run with multi app.